src/arrays/referencing_array_and_its_elements/medium/q12/MainTest.java

```diff
@@ -0,0 +1,67 @@
+package arrays.referencing_array_and_its_elements.medium.q12;
+
+import global.BaseTest;
+import global.tools.CustomAssertions;
+import global.tools.TestOption;
+import global.variables.Clause;
+import global.variables.clauses.*;
+import global.variables.wrappers.Optional;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static global.utils.ArrayUtil.*;
+
+public class MainTest extends BaseTest {
+    // Java
+
+    public static int n = 35;
+
+    public Clause[] testSentence() {
+        TestOption.isInputTest = true;
+        TestOption.defaultInput = arrayToInput(generateRandomArray(-1000.0, 1000.0, n)) + " " + n;
```

Comment:
It doesn't fit with the question to be testing negative numbers since a price will never be negative